pmd-core/src/main/java/net/sourceforge/pmd/Report.java

```diff
@@ -177,30 +184,6 @@ public Throwable getError() {
     }
 
 
-    private static String keyFor(RuleViolation rv) {
-
-        return StringUtils.isNotBlank(rv.getPackageName()) ? rv.getPackageName() + '.' + rv.getClassName() : "";
-    }
-
-    /**
-     * Calculate a summary of violation counts per fully classified class name.
-     *
-     * @return violations per class name
-     */
-    public Map<String, Integer> getCountSummary() {
```

Comment:
Do we first need to deprecate that method before removing it? It has of course no replacement in PMD6. We use in summary html renderer though....

Reply:
Ok, just seen #2680 - I'll continue review there and come back...